OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs

```diff
@@ -65,11 +65,13 @@ public ExamAttachmentServiceTests()
         private static string GetRandomMessage() => new MnemonicString().GetValue();
 
         private static SqlException GetSqlException() =>
-         (SqlException)FormatterServices.GetUninitializedObject(typeof(SqlException));
+            (SqlException)FormatterServices.GetUninitializedObject(typeof(SqlException));
+
         private static IQueryable<ExamAttachment> CreateRandomExamAttachments() =>
             CreateExamAttachmentFiller(DateTimeOffset.UtcNow)
-                .Create(GetRandomNumber()).AsQueryable();
+                .Create(GetRandomNumber()).AsQueryable();               
```

Comment:
Take away the extra space after the semi-colon

Reply:
Thanks for the review! How does anyone can identify such extra spaces on Visual Studio? Is there any extension, theme, or something else?

Reply:
Tools -> Options -> Text Editor -> General -> View White Spaces

![image](https://user-images.githubusercontent.com/89320816/140542954-b66f5256-73a5-442f-b536-0414fd94cd4b.png)

Reply:
Sincere thanks for the guidance! 
I was missing the important keyword - **whitespace**, hence failing to find out the desired solution from the internet!